lib/licensee/license.rb

```diff
@@ -64,6 +64,11 @@ def license_files
         @license_files ||= Dir.glob("#{license_dir}/*.txt")
       end
 
+      def spdx_dir
+        dir = ::File.dirname(__FILE__)
+        ::File.expand_path '../../vendor/license-list-XML/src', dir
```

Comment:
```suggestion
        ::File.expand_path '../../vendor/license-list-XML/src', __dir__
```

Reply:
Thanks! IIUC, the preceding line is now superfluous.